src/mnemonics.py

```diff
@@ -1,23 +1,108 @@
 from collections import namedtuple
 
+"""
+reg - One of 16 possible register locations ranging from s0 through sF or specified as a literal
 
-LOAD = namedtuple('LOAD', ('reg', 'reg_or_val'))
+aaa - 10-bit address, specified either as a literal or a three-digit hexadecimal value ranging
+      from 000 to 3FF or a labeled location
 
-EQU = namedtuple('EQU', ('alias', 'arg'))
-ORG = namedtuple('ORG', ('addr',))
+kk - 8-bit immediate constant, specified either as a literal or a two-digit hexadecimal value ranging
+     from 00 to FF or specified as a literal
+
+pp - 8-bit port address, specified either as a literal or a two-digit hexadecimal value ranging from 00 to FF
+      or specified as a literal
+
+ss - 6-bit scratchpad RAM address, specified either as a literal or a two-digit hexadecimal value
+     ranging from 00 to 3F or specified as a literal
+"""
+
+""" INSTRUCTIONS """
+
+ADD1 = namedtuple('ADD', ('reg', 'kk'))
+ADD2 = namedtuple('ADD', ('reg', 'reg2'))
```

Comment:
Hmm... Is there any good reason to have two instructions for two usages of particular instruction?

Imho it would be much better to have one instruction, with fields: `reg` and `reg_or_val` or `id_or_const`.